config/routes.rb

```diff
@@ -126,6 +126,10 @@
   end
   # get 'simulator/embed_cross/:id', to: 'simulator#embed_cross', as: 'simulator_embed_cross'
 
+  # vue simulator
+  get "/simulatorvue/*path", to: "vuesimulator#simulatorvue"
```

Comment:
Can we follow a consistent naming, in [here](https://github.com/CircuitVerse/CircuitVerse/blob/b67e61faf6c4119799b0f75120c49a0de4a66ff4/app/controllers/vuesimulator_controller.rb#L6) mentioned `simulatorvue`